Citizen-Science/src/screens/SplashScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { View, ImageBackground, Dimensions } from 'react-native';
+import { useNavigation, NavigationContainerRef } from '@react-navigation/native';
+
+// Get full width and height of the device's screen
+const screenWidth = Dimensions.get('window').width;
+const screenHeight = Dimensions.get('window').height;
+
+function SplashScreen() {
+    const navigation = useNavigation<NavigationContainerRef<any>>();
+
+    useEffect(() => {
```

Comment:
I would recommend updating this useEffect to:

```
useEffect(() => {
        // Splash screen for 2 seconds
	    const splashTimer = setTimeout(() => {
		    navigation.dispatch(
			    CommonActions.reset({
				    index: 0,
				    routes: [{ name: 'Welcome' }],
			    })
		    );
	    }, 2000);

        return () => clearTimeout(splashTimer); // This will cancel our splash screen after 2 seconds
    }, []);
```
This because if a user uses the back button at the Welcome Screen it will navigate them to the Splash Screen indefinitely